### PR TITLE
AppService Diagnostics Settings depend on the Tier

### PR DESCRIPTION
**Pull request by ReneHezser**
## Overview/Summary

Only AppServices with a Premium plan have the Diagnostic Settings AppServiceAntivirusScanAuditLogs and AppServiceFileAuditLogs available. This is taken into account in the policy "Deploy-Diagnostics-Website".

Addressing stories: [AB#26316](https://dev.azure.com/CSUSolEng/Azure%20Landing%20Zones/_workitems/edit/26316/)
Addressin issue: #1169 

## This PR fixes/adds/changes/removes

1. deploys a reduced diagnostic setting to non-premium AppServices

### Breaking Changes

n/a

## Testing Evidence

The policy definition has been testen on a basic and a premium AppService plan.
image

### Testing URLs

n/a

#### Azure Public

n/a

#### Azure US Gov (Fairfax)

n/a

## As part of this Pull Request I have

- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [x] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [x] Performed testing and provided evidence.
- [x] Ensured [contribution guidance](https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Contribution-Guide) is followed.
- [x] Updated relevant and associated documentation.
- [x] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

**Comment by ReneHezser** [comment]
> Thanks @ReneHezser can you please update the [`docs/wiki/Whats-new.md`](https://github.com/Azure/Enterprise-Scale/blob/main/docs/wiki/Whats-new.md) page with a brief note on the change here.
> 
> Thanks
> 
> Jack

I missed that part ¯\\_(ツ)_/¯

File: docs/wiki/Whats-new.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ Here's what's changed in Enterprise Scale/Azure Landing Zones:
 
 ### February 2023
 
+#### Policy
+
+- Updated `Deploy-Diagnostics-Website.json` policy to set Diagnostic Settings based on the AppService plan, as the Premium Tier has more settings available.
```
Comment (jtracey93) [inline docs/wiki/Whats-new.md:57]:
```suggestion
- Updated `Deploy-Diagnostics-Website.json` policy to set Diagnostic Settings based on the AppService plan, as the Premium Tier has more categories available.
```

**Review by jtracey93 — APPROVED** [review]
LGTM

**Comment by jtracey93** [comment]
Test ran fine, manually cleaning up 

image


related to bug #1199

**Comment by cveld** [comment]
When will this fix land in a new release of https://github.com/Azure/terraform-azurerm-caf-enterprise-scale?

**Comment by MarcelHeek** [comment]
Awaiting the answer as well

**Comment by ReneHezser** [comment]
Good news @cveld @MarcelHeek. The [PR ](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/pull/622) has been merged.

**Comment by jtracey93** [comment]
> When will this fix land in a new release of [Azure/terraform-azurerm-caf-enterprise-scale](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale)?

When we cut a new release, which i think one is in the pipeline @matt-FFFFFF, right?

**Comment by matt-FFFFFF** [comment]
We are creating a new release now

**Pull request merged**